deploy/csi-operator.yaml

```diff
@@ -981,7 +1000,12 @@ spec:
           hostPath:
             path: /var/lib/kubelet/
             type: Directory
-        - name: iscsiadm-bin
+        - name: chroot-iscsiadm
+          configMap:
+            defaultMode: 0755
```

Comment:
why not 0555? we just need read and execute permission, do we need write also?

Reply:
I observed iscsiadm creating required files in `/etc/iscsi/nodes` and `/etc/iscsi/send_targets`. So I think it needs permission.

Reply:
@mittachaitu this is the iscsiadm script that will be mounted inside the conatiner via configmap. We just need read and execute permission for a script to execute.

Reply:
Ah got it(We can change to 555).

Reply:
Yes 555 should work fine ... let me try that once .